Userland/Libraries/LibVideo/VP9/Decoder.cpp

```diff
@@ -1097,7 +1097,7 @@ inline i32 Decoder::round_2(T value, u8 bits)
 
 inline bool check_bounds(i64 value, u8 bits)
 {
-    const i64 maximum = (1u << (bits - 1u)) - 1u;
+    const i64 maximum = (1ll << (bits - 1ll)) - 1ll;
```

Comment:
Maybe convert this to east-const as well? (Which is the project's style of const)

Reply:
Ah, I'm surprised the formatter didn't take care of that like it did with a few others I noticed.. Fixed that!